tests/Shared/SolutionTests.proj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.Build.Traversal">
+  <!--
+    This file is used to build all the eligible test projects in the repo.
+    It is used to execute the tests in the outer loop and in the CI pipeline.
+
+    Generally, this project should invoked in the following way:
+
+      ./build.cmd -projects ./tests/Shared/SolutionTests.proj -restore -build -test /p:RunQuarantinedTests=[true|false]
+
+   -->
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_TestProjectsToExclude Include="$(RepoRoot)tests\Shared\**\*Tests.csproj" />
+    <_TestProjectsToExclude Include="$(RepoRoot)tests\testproject\**\*Tests.csproj" />
+    <_TestProjectsToExclude Include="$(RepoRoot)tests\TestingAppHost1\**\*Tests.csproj" />
+
+    <!-- This runs in a separate job -->
+    <_TestProjectsToExclude Include="$(RepoRoot)tests\Aspire.Workload.Tests\**\*Tests.csproj" />
+
+    <_TestProjectsToInclude Include="$(RepoRoot)tests\Aspire.Hosting.Tests\*Tests.csproj" />
```

Comment:
I think we missed this. It should be including all the test projects, right?

Reply:
Bah... /facepalm